csi.proto

```diff
@@ -502,6 +510,34 @@ message ControllerServiceCapability {
     RPC rpc = 1;
   }
 }
+
+message ControllerExpandVolumeRequest {
+  // The ID of the volume to expand. This field is REQUIRED
+  string volume_id = 1;
+
+  // the current capacity of the volume in bytes.
+  // This field is OPTIONAL
+  int64 current_capacity_bytes = 2;
+
+  // This field is REQUIRED. This allows CO to specify the
+  // capacity requirements of the volume after expansion
+  CapacityRange capacity_range = 3;
```

Comment:
May I assume that this `capacity_range` should specify a greater capacity than that of the volume in its current state? Given that this is a range, does that imply that the "required" and "limit" sub-fields have a lower bound that's greater than the current capacity? If so, it may be worth documenting that here.